cedar-policy-validator/src/human_schema/test.rs

```diff
@@ -53,19 +59,14 @@ mod demo_tests {
         let src = r#"
         action "Foo" appliesTo { context: {} };
         "#;
-        let (schema, _) = SchemaFragment::from_str_natural(src).unwrap();
-        let foo = schema.0.get(&None).unwrap().actions.get("Foo").unwrap();
-        assert_matches!(
-            foo,
-            ActionType {
-                applies_to: Some(ApplySpec {
-                    resource_types: None,
-                    principal_types: None,
-                    ..
-                }),
-                ..
-            }
-        );
+        match SchemaFragment::from_str_natural(src) {
+            Ok(_) => panic!("Should have failed to parse!"),
+            Err(e) => assert_matches!(e,
+                HumanSchemaError::Parsing(err) => assert_matches!(err.inner(),
+                    HumanSyntaxParseErrors::JsonError(errs) => assert!(errs.iter().any(|err| matches!(err, ToJsonSchemaError::NoPR { kind : PR::Resource, ..  })))
```

Comment:
Would prefer to use `expect_err` from `cedar_policy_core::test_utils` if that's not too much work. It's really helpful for keeping track of changes to error message output.